odoo/addons/base/models/ir_module.py

```diff
@@ -724,18 +724,18 @@ def get_values_from_terp(terp):
             'to_buy': False
         }
 
-    @api.model
-    def create(self, vals):
-        new = super(Module, self).create(vals)
-        module_metadata = {
-            'name': 'module_%s' % vals['name'],
+    @api.model_create_multi
+    def create(self, vals_list):
+        new_modules = super().create(vals_list)
```

Comment:
The name `new_modules` might be confusing when you think about "new" records.
Why not using `modules`?